bricksrc/quantities.py

```diff
@@ -1,109 +1,565 @@
-            "PM25_Level": {},
-            "TVOC_Level": {},
+        "subconcepts": {
+            "CO2_Level": {
+                QUDT.applicableUnit: [UNIT.PPM],
+                QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M0H0T0D1"],
+                SKOS.definition: Literal("The concentration ratio of some substance"),
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("Concentration"),
+                SKOS.broader: QUDTQK.Dimensionless,
+            },
+            "PM10_Level": {
+                QUDT.applicableUnit: [UNIT.PPM],
+                QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M0H0T0D1"],
+                SKOS.definition: Literal("The concentration ratio of some substance"),
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("Concentration"),
+                SKOS.broader: QUDTQK.Dimensionless,
+            },
+            "PM25_Level": {
+                QUDT.applicableUnit: [UNIT.PPM],
+                QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M0H0T0D1"],
+                SKOS.definition: Literal("The concentration ratio of some substance"),
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("Concentration"),
+                SKOS.broader: QUDTQK.Dimensionless,
+            },
+            "TVOC_Level": {
+                QUDT.applicableUnit: [UNIT.PPM],
+                QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M0H0T0D1"],
+                SKOS.definition: Literal("The concentration ratio of some substance"),
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("Concentration"),
+                SKOS.broader: QUDTQK.Dimensionless,
+            },
         },
     },
-    "Angle": {},
-    "Conductivity": {},
-    "Capacity": {},
+    "Angle": {OWL.sameAs: QUDTQK["Angle"]},
+    "Conductivity": {OWL.sameAs: QUDTQK["Conductivity"]},
+    "Capacity": {OWL.sameAs: QUDTQK["Capacity"]},
     "Enthalpy": {
         SKOS.definition: Literal(
             "(also known as heat content), thermodynamic quantity equal to the sum of the internal energy of a system plus the product of the pressure volume work done on the system. H = E + pv, where H = enthalpy or total heat content, E = internal energy of the system, p = pressure, and v = volume. (Compare to [[specific enthalpy]].)"
         ),
+        OWL.sameAs: QUDTQK["Enthalpy"],
+    },
+    "Mass": {
+        OWL.sameAs: QUDTQK["Mass"],
+        "subconcepts": {
+            "Grains": {
+                QUDT.applicableUnit: UNIT.GRAIN,
+                QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M1H0T0D0"],
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("Grains"),
+                SKOS.broader: QUDTQK.Mass,
+            }
+        },
+    },
+    "Phasor": {
+        "subconcepts": {
+            "PhasorAngle": {
```

Comment:
PhasorAngle is a subcomponent of Phasor. It does not mean that it is a subconcept of Phasor.